### DISCUSSION TITLE
Passwordless signin : FIDO2

### DISCUSSION THREAD

**Discussion by VaZark** [directus/directus#13163] [Feature Requests]
Looks like passwordless is all set to become the defacto standard for login. 

https://www.apple.com/newsroom/2022/05/apple-google-and-microsoft-commit-to-expanded-support-for-fido-standard/

Can FIDO2 support be considered for user login down the line ?

**Comment by benhaynes** [comment]
Absolutely.

**Reply by mmmaxxxe** [reply]
Would appreciate it a lot

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
